src/gui/properties/propertieswidget.h

```diff
@@ -97,11 +97,14 @@ protected slots:
     void showPiecesDownloaded(bool show);
     void showPiecesAvailability(bool show);
     void openSelectedFile();
+    void expandAll();
```

Comment:
It should be `expandFileList()` or something like this.
The same is about "collapse" counterpart.